maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -445,10 +479,13 @@ data class PressKeyCommand(
 
 data class EraseTextCommand(
     val charactersToErase: Int?,
+    val label: String? = null,
 ) : Command {
 
     override fun description(): String {
-        return if (charactersToErase != null) {
+        return if (label != null && label != "") {
```

Comment:
```suggestion
        return if (label != null) {
```

See my comment on `Condition.kt`